tests/test_slack.py

```diff
@@ -17,3 +56,107 @@ def test_slack_token(t: str) -> str:
     monkeypatch.setattr(settings, "slack_token", lambda: test_slack_token(token_value))
     notifier = slack.NotifierFactory.get_notifier()
     assert type(notifier) is want_type
+
+
```

Comment:
Could be good to have a test where we send nothing !

Reply:
Added!